packages/notifi-dapp-example/src/app/notifi/dashboard/page.tsx

```diff
@@ -41,17 +44,48 @@ export default function NotifiDashboard() {
 
   return (
     <div className="min-h-screen flex items-start flex-row">
+      {/* desktop sidebar */}
       <DashboardSideBar
         account={account}
         cardView={cardView}
         setCardView={setCardView}
+        className="md:block hidden"
       />
-      <div className=" flex flex-col grow h-screen ">
+      {/* mobile sidebar */}
+      {isSideNavOpen ? (
+        <DashboardSideBar
+          account={account}
+          cardView={cardView}
+          setCardView={setCardView}
+          setIsSideNavOpen={setIsSideNavOpen}
```

Comment:
1. I would name the useState name more consistently like `setIsSideBarOpen` to avoid confusion (They might think we have both navBar and sideBar if we named it this way).
2. In the component level,  I would not make the name like this shall we  just do like:
```
setIsOpen={setIsSideBarOpen}
```